components/style/values/specified/box.rs

```diff
@@ -177,3 +181,83 @@ impl Parse for WillChange {
         Ok(WillChange::AnimateableFeatures(custom_idents.into_boxed_slice()))
     }
 }
+
+#[cfg(feature = "gecko")]
```

Comment:
As a followup, we may want to not conditionally compile this, and just ensure they match, like `restyle_hints` and such.

Reply:
I'll check!